Weather Alert/CoreData/City+CoreDataProperties.swift

```diff
@@ -25,3 +25,20 @@ extension City {
     @NSManaged public var speed: NSNumber?
 
 }
+
+
+extension City {
+    
+    func dataToDictionary() -> [String : Any] {
```

Comment:
Are these values optional? I think they are.
Therefor, declaring return value as `[String : Any]` when the `Any` part may be nil would crash the app.